tools/data/ava/cut_videos.sh

```diff
@@ -29,6 +29,6 @@ for video in $(ls -A1 -U ${IN_DATA_DIR}/*)
 do
   out_name="${OUT_DATA_DIR}/${video##*/}"
   if [ ! -f "${out_name}" ]; then
-    ffmpeg -ss 900 -t 901 -r 30 -i "${video}" -strict experimental "${out_name}"
+    ffmpeg -ss 900 -t 901 -i "${video}" -strict experimental "${out_name}"
```

Comment:
try to move `-r 30` after `-i video`, as discussed here https://github.com/open-mmlab/mmaction2/issues/534#issuecomment-770386690